assets/lookbook.js

```diff
@@ -0,0 +1,121 @@
+document.addEventListener('DOMContentLoaded', function () {
+	const hotspots = document.querySelectorAll('.hotspot');
+	const lookbookImage = document.querySelector('.lookbook-image');
+	const productDetail = document.querySelector('.product-detail');
+	const closeButton = productDetail.querySelector('.product-card__close');
+	
+	hotspots.forEach(hotspot => {
+		hotspot.addEventListener('click', function (event) {
+			event.preventDefault();
+			
+			const productImage = productDetail.querySelector('.product-card__image');
```

Comment:
Porque está criando essas constantes TODA vez que o usuário clica no botão?

Reply:
Não tinha prestado atenção nisso. 
Transportei todas para o addEventListener.

`document.addEventListener('DOMContentLoaded', function () {
	const hotspots = document.querySelectorAll('.hotspot');
	const lookbookImage = document.querySelector('.lookbook-image');
	const productDetail = document.querySelector('.product-detail');
	const closeButton = productDetail.querySelector('.product-card__close');
	const productImage = productDetail.querySelector('.product-card__image');
	const productTitle = productDetail.querySelector('.product-card__title');
	const productPrice = productDetail.querySelector('.product-card__price');
	const productLink = productDetail.querySelector('.product-card__button');
	
	hotspots.forEach(hotspot => {
		hotspot.addEventListener('click', function (event) {
			event.preventDefault();
			
			const selectedImage = this.getAttribute('data-product-image');
			const selectedTitle = this.getAttribute('data-product-title');
			const selectedPrice = this.getAttribute('data-product-price');
			const selectedUrl = this.getAttribute('data-product-url');`